packages/ts/react-crud/src/autoform.tsx

```diff
@@ -220,6 +230,13 @@ export function AutoForm<M extends AbstractModel>({
       } else if (onSubmitSuccess) {
         onSubmitSuccess({ item: newItem });
       }
+      // Automatically clear the form after submitting a new item.
+      // Otherwise, there would be no way for the developer to clear it, as the
+      // there is no new value to set for the item prop to trigger the above
+      // effect in case the prop is already null, undefined or the empty item.
+      if (!item || item === emptyItem) {
+        form.clear();
+      }
```

Comment:
I just wonder if there should be a switch to disable this?

Reply:
What would be the use case? The only two I see is creating a new item, or editing an existing item. For the first one you want to clear the form, for the second one you assign the item prop.

Reply:
If there is no pressing need for having a switch off for this behavior, you can ignore this and merge.

Reply:
@taefi I think you need to remove your request for changes, or approve, otherwise merge is not available.